src/api/get-installations.js

```diff
@@ -59,6 +86,7 @@ async function findWindowsBrowsers () {
     if (edgeAlias)
         installations['edge'] = edgeAlias;
 
+
```

Comment:
Not here, i mean any application callback at background, which can be during addInstallation() execution.